contrib/builder/rpm/amd64/generate.sh

```diff
@@ -105,7 +105,7 @@ for version in "${versions[@]}"; do
 	case "$from" in
 		oraclelinux:7)
 			# Enable the optional repository
-			packages=( --enablerepo=ol7_optional_latest "${packages[*]}" )
+			packages=( "--enablerepo=ol7_optional_latest" "${packages[*]}" )
```

Comment:
I think it actually doesn't need to be quoted. Misinterpretation from Shellcheck here.